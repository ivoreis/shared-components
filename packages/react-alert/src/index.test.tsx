import React from 'react'
import { render } from '@testing-library/react'
import { Alert } from '.'

test('it works', () => {
  const { container } = render(<Alert kind="positive">My message</Alert>)
  expect(container.firstChild).toMatchSnapshot()
})
