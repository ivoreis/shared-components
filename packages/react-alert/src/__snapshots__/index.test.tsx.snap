// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<div
  class="css-1x8en36-Alert"
  kind="positive"
>
  My message
</div>
`;
